exporters/dsexporter/dsexporter.go

```diff
@@ -1,20 +1,46 @@
 package main
 
 import (
-	"fmt"
 	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-func ping(w http.ResponseWriter, req *http.Request) {
-	fmt.Fprintf(w, "pong")
+// CustomExporter is a dummy exporter that implements the prometheus.Collector interface.
+// The Custom exporter is a way to structure our code so that it can generate and expose metrics.
+type CustomExporter struct {
+	pingCounter prometheus.Counter
```

Comment:
With the current implementation, I think calling it `emptyCounter` is a better representation of what it does.
In any case, we should not have the word `ping` on the exporter-related files after this PR.

Reply:
@yftacherzog What do you mean by exporter-related files here?
quick question: since our focus should be on transitioning from a dummy service to a dummy exporter, I named it as `CustomExporter`

Reply:
I agree with @avi-biton about naming the type `CustomCollector` as it implements the `Collector` interface (https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Collector).

Reply:
Okay renamed it to `CustomCollector` and removed the `ping` keywords